.github/workflows/test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     - name: Version check
       run: |
         python otherfile/version_check.py
-      if: matrix.python-version == 3.8
+      if: matrix.python-version == 3.9
```

Comment:
I have a suggestion to prevent duplication:
We can define two environmental variables (like `TEST_PYTHON_VERSION` and `TEST_OS`) and use them.
Take a look at this [link](https://docs.github.com/en/actions/learn-github-actions/variables)